service/application/src/main/resources/logback-spring.xml

```diff
@@ -26,7 +26,7 @@
         <if condition='property("USE_JSON").equals("true")'>
             <then>
                 <encoder class="net.logstash.logback.encoder.LogstashEncoder">
-                    <customFields>{"component":"event-management-agent-${agentId}"}</customFields>
+                    <customFields>{"component":"event-management-agent-${agentId}", "ema_id":"${agentId}"}</customFields>
```

Comment:
Since the `agentId` is already mentioned in the `component` value, can we use that value instead?

Merging this change would mean, releasing a new EMA after running tests, updating all production C-EMAs. So it would be great if we can use the existing value from `component` instead.

Reply:
We absolutely can. That would have saved me a lot of time. I'll edit and re-test the monitor.